flow-service/src/config/resolution/service-config-accessor.ts

```
@@ -1,92 +1,135 @@
+          OPTIONAL { ?channel fsvc:logLevel ?level . }
+        }
+      }
+    `;
+    const enabledStr = await this.getConfigValueFromSparql(sparql, 'enabled');
+    const level = await this.getConfigValueFromSparql(sparql, 'level');
+    return { enabled: enabledStr === 'true', level: level ?? undefined };
+  }
+
+  private async getConfigValueFromSparql(sparql: string, variable: string): Promise<string | undefined> {
     if (!this.bundle.engine) {
       throw new Error('SPARQL engine not initialized in Quadstore bundle');
```

ESLint

[error] Strings must use doublequote.

(quotes)